packages/atomic/src/components/common/search-box/suggestions-common.ts

```diff
@@ -96,9 +139,9 @@ export interface SearchBoxSuggestionsBindings extends Bindings {
    */
   isStandalone: boolean;
   /**
-   * The [search box headless controller](https://docs.coveo.com/en/headless/latest/reference/search/controllers/search-box/).
+   * The search box headless controller.
    */
-  searchBoxController: SearchBox;
+  searchBoxController: SearchBoxController;
   /**
    * The amount of queries displayed when the user interacts with the search box, property set on the [Atomic Search box](https://docs.coveo.com/en/atomic/latest/reference/components/atomic-search-box/#properties).
```

Comment:
Out of scope, but since I spotted it...

```suggestion
   * The number of queries to display when the user interacts with the search box.
```